src/components/Chat/BottomSheetAttachment.js

```diff
@@ -0,0 +1,105 @@
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <MediasIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send Photos & Videos</Text>
+          </View>
+          {props.isLoadingUploadMedia ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+      <TouchableNativeFeedback onPress={() => props.onOpenGIF()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <GIFIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send a GIF</Text>
+          </View>
+          {props.isLoadingUploadGIF ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+      <TouchableNativeFeedback onPress={() => props.onOpenCamera()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <CameraIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Take a Picture</Text>
+          </View>
+          {props.isLoadingUploadCamera ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+
+      <TouchableNativeFeedback onPress={() => props.onOpenFile()}>
+        <View
+          style={[
+            styles.card,
+            {
+              borderBottomWidth: 1,
+              borderBottomColor: colors.alto
+            }
+          ]}>
+          <View style={styles.wrapCardImage}>
+            <FileIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send a File</Text>
+          </View>
+          {props.isLoadingUploadFile ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+    </View>
+  </BottomSheet>
+));
+BottomSheetAttachment.displayName = 'BottomSheetAttachment';
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: 0,
+    paddingTop: 0,
+    paddingBottom: 0
+  },
+  containerBottomSheet: {
+    marginTop: 20
+  },
+  card: {
+    height: 52,
+    paddingHorizontal: 20,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    borderTopWidth: 1,
+    borderTopColor: colors.alto
+  },
+  wrapCardImage: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  textCard: {
+    fontFamily: fonts.inter[500],
+    fontSize: 14,
+    color: colors.black,
+    paddingLeft: 9
+  }
+});
+
+BottomSheetAttachment.propTypes = {
+  onOpenMedia: PropTypes.func,
+  isLoadingUploadMedia: PropTypes.bool,
+  onOpenGIF: PropTypes.func,
+  isLoadingUploadGIF: PropTypes.bool,
+  onOpenCamera: PropTypes.func,
+  isLoadingUploadCamera: PropTypes.bool,
+  onOpenFile: PropTypes.func,
+  isLoadingUploadFile: PropTypes.bool
+};
+
+export default BottomSheetAttachment;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code looks good overall, but there's a potential issue with the `TouchableNativeFeedback` component. This component is specific to Android and may cause issues on iOS. Consider using a more cross-platform friendly component like `TouchableOpacity` or `Pressable`.
```diff
-       <TouchableNativeFeedback onPress={() => props.onOpenMedia()}>
+       <Pressable onPress={() => props.onOpenMedia()}>
```
Also, apply this change to all instances of `TouchableNativeFeedback` in your code.

Another point to consider is error handling for the functions passed as props. If any of these functions (`onOpenMedia`, `onOpenGIF`, `onOpenCamera`, `onOpenFile`) fail, it could lead to an unhandled exception. It would be better to ensure these functions are always provided and handle errors properly.
```diff
-       <Pressable onPress={() => props.onOpenMedia()}>
+       <Pressable onPress={() => { try { props.onOpenMedia() } catch (e) { console.error(e); } }}>
```
Again, apply this change to all instances where you're calling these functions.

Lastly, PropTypes validation can be improved by making necessary props required. For example, if `onOpenMedia` is a required function, it should be marked as such.
```diff
-   onOpenMedia: PropTypes.func,
+   onOpenMedia: PropTypes.func.isRequired,
```
Apply this change to all necessary props.


<!-- This is an auto-generated comment by OSS CodeRabbit -->